libs/platform/src/lib/components/form/combobox/directives/auto-complete.directive.ts

```diff
@@ -0,0 +1,134 @@
+import { Directive, ElementRef, EventEmitter, HostListener, Input, Output } from '@angular/core';
+import { BACKSPACE, CONTROL, DELETE, ENTER, ESCAPE, LEFT_ARROW, RIGHT_ARROW } from '@angular/cdk/keycodes';
+
+import { KeyUtil } from '@fundamental-ngx/core';
+import { OptionItem } from '../../../../domain/data-model';
+
+export interface AutoCompleteEvent {
+    term: string;
+    forceClose: boolean;
+}
+
+@Directive({
+    // tslint:disable-next-line:directive-selector
+    selector: '[fdp-auto-complete]'
+})
+export class AutoCompleteDirective {
```

Comment:
I see a lot of code is taken from core in this directive, what about creating 1 file, which will match requirements from both libraries?

Reply:
Let's create an issue as a follow up to refactor these two if possible.